### PR TITLE
Motion refactoring fixes

### LINKED ISSUE #4201: Motion detail faulty change behaviour
- [x] Motion create save button activation is delayed
- [x] Editoral final version save/delete behavior is currently odd

### PR DESCRIPTION
**Pull request by bastianjoel**
resolves #4201

### CODE REVIEW

File: client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/pages/motion-view/components/motion-content/motion-content.component.ts [hunk 1]
```diff
@@ -131,21 +143,27 @@ export class MotionContentComponent extends BaseMotionDetailChildComponent {
...
         this.formattedTextPlain$ = combineLatest([
             this.meetingSettingsService.get(`motions_line_length`),
-            this.unifiedChanges$
+            this.unifiedChanges$,
+            this.updateFormatedText.pipe(startWith(undefined))
         ]).pipe(
             map(([lineLength, changes]) => {
                 if (lineLength) {
-                    return this.motionFormatService.formatMotion({
+                    const text = this.motionFormatService.formatMotion({
```
Comment (reiterl) [inline client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/pages/motion-view/components/motion-content/motion-content.component.ts:158]: Why this const, surplus?

**Review by reiterl — APPROVED** [review]
Code seems okay, as far as I know. One small remark.

File: client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/pages/motion-view/components/motion-content/motion-content.component.ts [hunk 2]
```diff
@@ -131,21 +143,27 @@ export class MotionContentComponent extends BaseMotionDetailChildComponent {
...
                         targetMotion: this.motion,
                         crMode: this.changeRecoMode,
                         changes: this.changeRecoMode === ChangeRecoMode.Original ? [] : changes,
                         lineLength: this.lineLength,
                         highlightedLine: this.highlightedLine,
                         firstLine: this.motion.firstLine
                     });
+                    return text;
```
Comment (bastianjoel) [inline client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/pages/motion-view/components/motion-content/motion-content.component.ts:166]:
```suggestion
                    return this.motionFormatService.formatMotion({
                        targetMotion: this.motion,
                        crMode: this.changeRecoMode,
                        changes: this.changeRecoMode === ChangeRecoMode.Original ? [] : changes,
                        lineLength: this.lineLength,
                        highlightedLine: this.highlightedLine,
                        firstLine: this.motion.firstLine
                    });
```

**Review by emanuelschuetze — APPROVED** [review]
Looks good

**Pull request merged**